ci/kubetest/test_clickhouse_different_image.py

```diff
@@ -14,7 +14,7 @@
 
 clickhouse:
   image:
-    tag: 22.3.6.5-alpine
+    tag: 22.3.13.80-alpine
```

Comment:
but at that point this is not anymore a different image, making the test useless

Reply:
It's the alpine version though, rather than the standard? Previously we were using 22.3.6.5 everywhere, and 22.3.6.5-alpine here

Reply:
ahhhh sorry 🤦

Reply:
no worries! appreciate the speedy review + double checking 🎉